validation_enums.node.js

```diff
@@ -0,0 +1,554 @@
+module.exports = {
```

Comment:
We were unable to retrieve these from the schema itself?

Reply:
So, we can, but will require dot walking quite a ways (e.g. `properties.building.properties.zone.properties.zone_wall.items.properties.wall_assembly_code.enum`). I had originally started by porting them from the constants in validation_rules.js (if I remember right) to clean that up and prevent circular dependencies.

Reply:
I think I'm generally fine with that, and maybe just keeping this file as a generic wrapper for them.  As much as possible, we should keep the schema as a single source of truth